submissions/Nikita_Lemesiev/task_js_dom/scripts.js

```diff
@@ -0,0 +1,100 @@
+const series = [
+    ['popular',
+    'Разберитесь с деньгами',
+    '<img id="main-img" src="images/money-problems.png">',
+    'Финансовые консультанты делятся простыми советами о том, как меньше тратить и больше экономить, чтобы контролировать свои средства и достигать задуманного.'],
+
+    ['detective',
+    'Женщина живущая в доме напротив девушки в окне',
+    '<img id="main-img" src="images/woman-next-door.png">',
+    'В перерывах между вином, таблетками и запеканками впечатлительная Анна влюбляется в красавчика-соседа и становится свидетельницей убийства. Или ей показалось?'],
+
+    ['comedy',
+    'В лучшем мире',
+    '<img id="main-img" src="images/goodplace.png">',
+    'Элеанор умирает и оказывается в раю, в который попадают только очень нравственные люди. Она понимает, что ее приняли за кого-то другого.'],
+
+    ['criminal',
+    'Линкольн для адвоката',
+    '<img id="main-img" src="images/lincolnlawyer.png">',
+    'Микки Холлер, успешный адвокат из Лос-Анджелеса, оказавшийся вне игры после аварии, берется за дело об убийстве, чтобы вернуться в строй и за руль фирменного «Линкольна».'],
+
+    ['fantacy',
+    'Архив 81',
+    '<img id="main-img" src="images/archiv81.png">',
+    'Архивиста, восстанавливающего поврежденные видеозаписи, затягивает в водоворот мистических событий, связанных с исчезновением режиссера и деятельностью сатанинской секты.'],
+
+    ['anime',
+    'Дядя из другого мира',
+    '<img id="main-img" src="images/another-uncle.png">',
+    'После 17 лет в коме дядя Такафуми внезапно просыпается в зрелом возрасте и начинает говорить на неизвестном языке, демонстрируя магические способности.'
+    ],
+
+    [ 'main',
+    'Кинозал Kottans',
+    '<img id="main-img" src="images/cinema.jpg">',
+    'Не знаешь что посмотреть - добро пожаловать в рандомный подбор от студента Kottans'
+    ]
+
+];
+
+//Изменяет популярное
+function popular(event) {
+    document.getElementById("title").textContent = series[0][1];
+    document.getElementById("image").innerHTML = series[0][2];
+    document.getElementById("description").textContent = series[0][3];
+};
```

Comment:
Don't repeat yourself (DRY).
Create one function which creates elements and adds to the wrap.
Create a dom element, not update his properties and values.